frontend/src/components/VDetailOrganisation.vue

```diff
@@ -1,18 +1,16 @@
 <template>
   <div>
-    <b-list-group>
-      <b-list-group-item class="header-item active">
-        <icon class="mb-1" name="users"/>
+      <h4 class="card-title ml-2">
+        <icon class="mb-1 mr-1" name="users"/>
         {{result.name}}
-      </b-list-group-item>
-    </b-list-group>
+      </h4>
 
     <div v-if="result.locations.length" class="card mt-2 mb-2">
       <div class="card-body">
-        <b-list-group-item variant="dark">
+        <b-list-group-item class="mb-2 bg-light">
           {{ $t('contact_info') }}
         </b-list-group-item>
-        <b-list-group-item v-if="result.locations && result.locations.length" v-for="(location, idx) in result.locations" :key="Object.keys(location)[idx]">
+        <b-list-group-item v-for="(location, index) in result.locations" :key="Object.keys(location)[index]">
```

Comment:
Perhaps I wasn't clear. I meant use `index` as key, as in:
`<b-list-group-item v-for="(location, index) in result.locations" :key="index">`
because it's not really clear what `Object.keys(location)[index]` means